components/breadcrumb/src/vwc-breadcrumb.ts

```diff
@@ -0,0 +1,12 @@
+import '@vonage/vvd-core';
+import '@vonage/vwc-icon';
+import '@vonage/vivid/breadcrumb/index.js';
+
+const style = document.createElement('style');
+style.innerHTML = `
+	vwc-breadcrumb {
+		--icon-size: 14px;
+	}
+`;
+document.head.appendChild(style);
```

Comment:
why?

Reply:
We need to set the `icon`'s size inside the element. 
Since we have no access to the `scss` file and actually have no `scss` file, this is the way I thought of doing it.
Got a suggestion?